river/command/filter.zig

```diff
@@ -27,19 +27,31 @@ const ViewStack = @import("view_stack.zig").ViewStack;
 const Error = @import("../command.zig").Error;
 const Seat = @import("../Seat.zig");
 
+const FilterKind = enum {
+    @"app-id",
+    title,
+};
+
 pub fn floatFilterAdd(
     allocator: *mem.Allocator,
     seat: *Seat,
     args: []const [:0]const u8,
     out: *?[]const u8,
 ) Error!void {
-    if (args.len < 2) return Error.NotEnoughArguments;
-    if (args.len > 2) return Error.TooManyArguments;
+    if (args.len < 3) return Error.NotEnoughArguments;
+    if (args.len > 3) return Error.TooManyArguments;
+
+    const kind = std.meta.stringToEnum(FilterKind, args[1]) orelse return Error.UnknownOption;
+    const map = switch (kind) {
+        FilterKind.@"app-id" => &server.config.float_filter_app_ids,
+        FilterKind.title => &server.config.float_filter_titles,
```

Comment:
We can use enum literals here to keep things a bit more concise:
```suggestion
        .@"app-id" => &server.config.float_filter_app_ids,
        .title => &server.config.float_filter_titles,
```